fieldmanager.php

```diff
@@ -3,22 +3,22 @@
  * Fieldmanager Base Plugin File.
  *
  * @package Fieldmanager
- * @version 1.2.3
+ * @version 1.2.4
  */
 
 /*
 Plugin Name: Fieldmanager
 Plugin URI: https://github.com/alleyinteractive/wordpress-fieldmanager
 Description: Add fields to WordPress programatically.
 Author: Alley Interactive
-Version: 1.2.3
+Version: 1.2.4
 Author URI: https://www.alleyinteractive.com/
```

Comment:
Maybe we should update `Author` and `Author URI` while we're here?

Reply:
Great idea!  I've updated that.